maitake-sync/src/wait_cell.rs

```diff
@@ -481,13 +481,10 @@ impl WaitCell {
         }
     }
 
-    // TODO(eliza): is this an API we want to have?
-    /*
     /// Returns `true` if this `WaitCell` is [closed](Self::close).
-     pub(crate) fn is_closed(&self) -> bool {
-       self.current_state() == State::CLOSED
+    pub fn is_closed(&self) -> bool {
```

Comment:
super tiny nit: since this has no side effects, i usually like to stick a on it
```suggestion
    #[must_use]
    pub fn is_closed(&self) -> bool {
```
to indicate that. it could probably also have an `#[inline]` attribute but :woman_shrugging: